general/odoo_basics/add_user.rst

```diff
@@ -1,46 +1,141 @@
-=================
-How to add a user
-=================
+==================================
+Add Users and Manage Access Rights
+==================================
 
-Odoo provides you with the option to add additional users at any given
-point.
+Odoo defines a *user* as someone who has access to a database to perform daily tasks. You can add as
+many users as you need, and in order to restrict the type of information each user can access, rules
+can be applied. Users and access rights can be added and changed at any point.
+
+.. _general/odoo_basics/add_individual_user:
 
 Add individual users
 ====================
 
-From the Settings module, go to the submenu :menuselection:`Users --> Users` and click on
-**CREATE.** First add the name of your new user and the professional
-email address - the one he will use to log into Odoo instance - and a
-picture.
+Go to :menuselection:`Settings --> Manage Users` and click on *Create*.
+
+.. image:: media/manage_users.png
+   :align: center
+   :height: 280
+   :alt: View of the settings page emphasizing the manage users field in Odoo
 
-.. image:: media/add_user01.png
-    :align: center
+| Fill in the form with the needed information. Under the tab
+  :ref:`Access Rights <general/odoo_basics/add_user/access_rights>` choose the group within
+  each application the user can have access.
+| The list of applications shown is based on the applications you have installed on the database.
```

Comment:
```suggestion
| The list of applications shown is based on the applications installed on the database.
```

Reply:
Change this, you have is not needed and might not be correct.